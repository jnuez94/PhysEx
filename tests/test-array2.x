int *arr;
function main () {
	arr = [5];
	arr[0] = 42;
  arr[1] = 24;
  arr[2] = 42;

	printi(arr[0]);
  print(" ");
  printi(arr[1]);
  print(" ");
  printi(arr[2]);
}
