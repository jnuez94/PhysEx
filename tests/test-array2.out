42

24

42
